console/jsre/deps/web3.js

```diff
@@ -5340,15 +5340,19 @@ var Iban = require('../iban');
 var transfer = require('../transfer');
 
 var blockCall = function (args) {
-    return (utils.isString(args[0]) && args[0].indexOf('0x') === 0) ? "klay_getBlockByHash" : "klay_getBlockByNumber";
+    return (utils.isString(args[0]) && args[0].indexOf('0x') === 0 && args[0].length === 66) ? "klay_getBlockByHash" : "klay_getBlockByNumber";
```

Comment:
This is minor comment, but do you think thiat checking `args[0].length === 66` here is appropriate? For example, `klay_getBlock("0x0")` calls `klay_getBlockByNumber`. What do you think?

Reply:
Thanks for good review!
I have a question.
Could the hash value be `"0x0"` not `"0x000...000"`...?

Reply:
I meant wrong hash value. The correct value of hash is `0x000...000`, but if we put `0x0` to `getBlock` then it calls `getBlockByNumber`. What do you think about this?

Reply:
Not only blockCall, BlockNumberOrHash is also divided based on whether it's length is 66. (Ethereum same)
So I think .. even if the user intended blockhash, `0x0` should be considered a blocknumber in code.